src/Modules/UngradedModules.jl

```diff
@@ -6361,6 +6362,10 @@ function free_resolution(M::SubquoModule{<:MPolyRingElem};
     res = Singular.fres(singular_kernel_entry, length, "complete")
   elseif algorithm == :sres
     res = Singular.fres(singular_kernel_entry, length)
+  elseif algorithm == :mres
```

Comment:
Don't these also require a recent enough Singular.jl? If so then Project.toml should be adjusted, too

Reply:
Sure! I am not sure I am doing this correctly in https://github.com/oscar-system/Oscar.jl/pull/2500/commits/e72cb47fb35fe8d53dbb9deee7dec1d77582bf52, do I just by hand bump the compat entry in Project.toml? Never had to fiddle around with this before.